learningresources/models.py

```diff
@@ -61,6 +61,7 @@ class StaticAsset(BaseModel):
     Holds static assets for a course (css, html, javascript, images, etc)
     """
     course = models.ForeignKey(Course)
+    path = models.TextField()  # path in static folder
```

Comment:
It added it because I didn't know you could query the `.name` in a filter, since it's not the fieldname. From a StackOverflow posting, it appears that the ORM uses the name as the filter.